pkg/podexecutor/staticpod.go

```diff
@@ -41,17 +41,65 @@ var (
 	defaultAuditPolicyFile = "/etc/rancher/rke2/audit-policy.yaml"
 )
 
+type ControlPlaneResources struct {
```

Comment:
We can probably drop all of the prefixed "kube" since that's implied and will make visually scanning this struct a lot easier.

Reply:
I am +1 on keeping them as-is for parity with the Image stuff at https://github.com/rancher/rke2/blob/6c1df6c9f38c89f4c9d629a2f775804f589795b7/pkg/images/images.go#L47-L52